test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -764,6 +764,7 @@ def test_qat_per_channel_weight_custom_dtype(self):
 
 
 # TODO: enable this in the next PR
+# TODO: fix for 3.12
 @skipIfNoQNNPACK
 @unittest.skipIf(
     sys.version_info >= (3, 12), "torch.compile is not supported on python 3.12+"
```

Comment:
Don't you want to update a message here as well
```suggestion
    sys.version_info >= (3, 12), "Broken in python 3.12 at the moment, see url-to-issue"
```